styles/nav.scss

```diff
@@ -81,6 +81,7 @@ $daptiv-subnav-tab-h-pad: 14px;
         padding-right: $standard-whitespace-unit;
         text-decoration: none;
         text-overflow: ellipsis;
+        overflow: hidden;
```

Comment:
[No fix required/question]: Do we want to move towards having CSS properties in alphabetical order (to aid subsequent reading/groking]?